com.chartboost.mediation.demo.upm/Assets/Plugins/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.unity3d.player" xmlns:tools="http://schemas.android.com/tools" android:name="androidx.multidex.MultiDexApplication">
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+  <uses-permission android:name="android.permission.READ_PHONE_STATE" />
+  <application android:debuggable="true">
+    <activity android:name="com.unity3d.player.UnityPlayerActivity" android:theme="@style/UnityThemeSelector">
+      <intent-filter>
+        <action android:name="android.intent.action.MAIN" />
+        <category android:name="android.intent.category.LAUNCHER" />
+      </intent-filter>
+      <meta-data android:name="unityplayer.UnityActivity" android:value="true" />
+      <meta-data android:name="unityplayer.ForwardNativeEventsToDalvik" android:value="true" />
+    </activity>
+    <meta-data android:name="com.google.android.gms.ads.APPLICATION_ID" android:value="ca-app-pub-6548817822928201~9780573322" />
+    <meta-data android:name="applovin.sdk.key" android:value="wy1dIpr2F2-9fqTT2ihBprfAvPyfd6qm_81WkbsZC6Fl31vaW4aiDFxR52R89x98WWNfPmTIGMInIvffFTOxLz" />
```

Comment:
This 2 fields should not be commited, the AndroidManifest in the demo apps should contain a clean version of it so people can use whatever keys/setup they need.

Reply:
oh yeah this was by mistake